raiden_libs/test/fixtures/web3.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import pytest
+import gevent
+
+import rlp
+from _pytest.monkeypatch import MonkeyPatch
+from eth_utils import decode_hex, denoms
+from ethereum.transactions import Transaction
+from eth_tester import EthereumTester, PyEVMBackend
+
+
+from web3 import Web3
+from web3.providers.eth_tester import EthereumTesterProvider
+from web3.contract import Contract
+from web3.providers.rpc import HTTPProvider
+
+from raiden_libs.utils import (
+    address_from_signature,
+    keccak256,
+)
+# import microraiden.utils.contract
```

Comment:
remove commented out code

Reply:
The commented out code is still there @pcppcp

Reply:
@pcppcp this is still here ;)

Reply:
ah damn to hell with that sneaky comment